docs/nasa_ads/nasa_ads.rst

```diff
@@ -21,6 +25,21 @@ Search works by specific identifier
 .. code-block:: python
 
     from astroquery import nasa_ads as na
+
+    # if you don't store your token as an environment variable
```

Comment:
This is very much the subject of a follow-up PR, but it would be great to have more, and full working examples.